devops/infrastructure/bash-config.sample.cfg

```diff
@@ -5,7 +5,8 @@ REGION=us-east-1
 CLI_PROFILE=joystream-user
 KEY_PATH="/Users/joystream/Joystream/joystream-key.pem"
 AWS_KEY_PAIR_NAME="joystream-key"
-EC2_INSTANCE_TYPE=t2.micro
+NODE_EC2_INSTANCE_TYPE=t2.micro
```

Comment:
I see this is the type used for the "InstanceLaunchTemplate", and RPCInstance doesn't specify the instance type so its taken from the template. Therefore maybe a better name to reflect this, like `DEFAULT_EC2_INSTANCE_TYPE`

Reply:
Added DEFAULT_EC2_INSTANCE_TYPE